examples/examples.json

```diff
@@ -63,10 +63,6 @@
                     "title": "Viewport based styling",
                     "file": "styling/viewport.html"
                 },
-                {
```

Comment:
I see that we remove zoom example, why?. Should we have another one with `zoomRange`?

Reply:
The zoom example was actually being similar to `scaled` not to `zoomRange` actually. We could add an example, though

Reply:
It would be nice to add this example ☝️ ✨